README.md

```diff
@@ -3,39 +3,83 @@
 
 ```swift
 var age = 19
 
+let isOfDrinkingAge = age > 21 // false, because 19 is not greater than 21
+```
+
+As was briefly mentioned in that lesson, those expressions become powerful using *conditional statements* which allow you to write pieces of code which can be executed (or not executed) based on certain conditions. These are called *control flow* constructs, because they allow you to control the execution flow of your program, allowing your program's output to react to conditions, instead of always providing the same output.
+
+## If statement
+
+The `if` statement makes the execution of a set of statements (also called a *block of code*) following it conditional based on a `Bool` value or expression. The given block of code will be executed *only if* the expression evaluates to `true`:
+
+```swift
+let age = 19
+
 if age < 21 {
-    print("You're not allowed to drink!")
+	print("You are not allowed to drink, yet.")
 }
 
-// prints "You're not allowed to drink!"
+// prints "You are not allowed to drink, yet."
 ```
 
-* Explain that an *if* statement can provide an alternative set of statements, known as an *else clause*, for situations when the *if* condition is *false*.
+You can type this block of code in a new Playground file and experiment with giving `age` different values. You will notice that the `print` statement will no longer be executed once you increase the value of `age` to 21 or higher. 
+
+Now try to extend the program by adding a second `if` statement which prints a message if the `age` is greater than or equal to 21. 
+
+You will have ended up with something like this:
 
 ```swift
-var myHeight = 50
+let age = 55
+
+if age < 21 {
+	print("You are not allowed to drink, yet.")
+}
+
+if age >= 21 {
+	print("You are old enough to drink. 🍻")
+}
 
-if myHeight > 60 {
-    print("You're free to go on the ride, enjoy!")
+// prints "You are old enough to drink. 🍻"
+```
+
+## Else clause
+
+This seems hard to read and will also lead to duplicated work should the condition ever change in the future. Thankfully, Swift provides a better way to provide an alternative block of code to an `if` statement, an *else clause*:
+
+```swift
+let age = 55
+
+if age < 21 {
+	print("You are not allowed to drink, yet.")
 } else {
-    print("I'm sorry, you're too short to go on this ride today.")
+	print("You are old enough to drink. 🍻")
 }
 
-// prints "I'm sorry, you're too short to go on this ride today."
+// prints "You are old enough to drink. 🍻"
 ```
 
-* Create/chain multiple *if* statements together to consider additional clauses:
+The code block following an `else` statement will only be executed if the condition provided to the corresponding `if` statement is `false`.
+
+## Nested conditionals
+
+Sometimes the execution flow of a program is not as clear cut as an either-or, there may be a third option or a default case. If that happens, you can chain multiple if-else statements together like this:
 
```

Comment:
I like the idea of humanizing this a bit and having them visualize the problem they are solving here stating something as follows. If the temperature outside is under 32 degrees fahrenheit then I will need to put on a heavy coat, if it's over 90 degrees then I will make sure to wear shorts. If it's anywhere in between then I will wear pants. I can't solve this problem with just one `if-else` statement. Instead of having three if statements back to back to back, we know this is not the best approach. Instead we can nest conditionals and produce something like the following: 

```swift
let temperatureInFahrenheit = 100

if temperatureInFahrenheit < 32 {
    print("Snow! 🌨, put on a heavey coat.")
} else if temperatureInFahrenheit >= 90 {
    print("Very hot! ☀️, make sure to wear shorts!")
} else {
    print("Just right. 🌻, wear pants today.")
}

// prints "Very hot! ☀️, make sure to wear shorts!"
```